system-tests/tests/smoke/cre/por_price_provider.go

```diff
@@ -42,9 +42,12 @@ func setupFakeDataProvider(testLogger zerolog.Logger, input *fake.Input, authKey
 			return nil, fmt.Errorf("no expected prices not found for feedID: %s", feedID)
 		}
 
+		currentPrice := expectedPrices[*priceIndex]
+		testLogger.Info().Msgf("HTTP response for feedID %s - priceIndex: %d, currentPrice: %.2f", feedID, *priceIndex, currentPrice)
```

Comment:
is 2 enough of a precision? later on you use precision of 10

Reply:
changed to 10